go/engine/signup.go

```diff
@@ -234,12 +234,14 @@ func (s *SignupEngine) registerDevice(a libkb.LoginContext, ctx *Context, device
 		s.G().Log.Warning("error saving session file: %s", err)
 	}
 
+	// Create the secret store as late as possible here, as the username may
+	// change during the signup process.
 	if s.arg.StoreSecret {
-		// Create the secret store as late as possible here
-		// (instead of when we first get the value of
-		// StoreSecret) as the username may change during the
-		// signup process.
 		secretStore := libkb.NewSecretStore(s.G(), s.me.GetNormalizedName())
+		if secretStore == nil {
+			s.G().Log.Info("aborting StoreSecret: secretStore is nil")
+			return nil
+		}
```

Comment:
Standalone `signup` + `team create` workflow seems to work fine without that block, but I had a failing engine test on one of two runs (the second time it went through). Should I remove it and see if it passes CI?